gamemodes/attdef.pwn

```diff
@@ -13,6 +13,10 @@
 #include <sampac> 			// THE MIGHTY NEW ANTICHEAT
 #include <mSelection>       // Selection with preview models feature library
 #include <gBugFix>			// Fix false vehicle entry as passenger (G (teleport/distance) bug)
+#include <md-sort>          // Sorts multi dimensional arrays
+
+#define RELEASE_VERSION 1
+#define ROUND_NEVER_END 0   // Used for testing purposes only
```

Comment:
It's easy switch this before pressing compile button, but if you don't like it then sure.